### PR TITLE
Birthday

### PR DESCRIPTION
**Pull request by dfuentes**
Logs queue stats retrieved from the rabbitmq management API.  Currently logs:

- Total message count/rate
- Messages ready count/rate
- Messages unacknowledged count/rate
- Total published messages, publish rate
- Total acked messages, ack rate

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+type M map[string]interface{}
+
+func init() {
+	RabbitMQHost = getEnv("RABBITMQ_HOST")
+	RabbitMQPort = getEnv("RABBITMQ_PORT")
+	RabbitMQUser = getEnv("RABBITMQ_USER")
+	RabbitMQPass = getEnv("RABBITMQ_PASS")
+
+	pollIntervalRaw := getEnv("POLL_INTERVAL")
+	if pollIntervalRaw == "" {
```
Comment (nathanleiby) [inline main.go]: `getEnv` fails with a `log.Fatalf`, so it will exit before meeting this condition

Comment (dfuentes) [inline main.go]: ah, good catch, I copied that code from `marathon-stats` so its wrong there.

File: main.go [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+}
+
+func logQueueStats(queueInfo rabbithole.QueueInfo) {
+	logger.Gauge(fmt.Sprintf("%s.messages.count", queueInfo.Name), queueInfo.Messages)
+	logger.Gauge(fmt.Sprintf("%s.messages-ready.count", queueInfo.Name), queueInfo.MessagesReady)
+	logger.Gauge(fmt.Sprintf("%s.messages-unacknowledged.count", queueInfo.Name), queueInfo.MessagesUnacknowledged)
+	logger.Gauge(fmt.Sprintf("%s.publish.count", queueInfo.Name), queueInfo.MessageStats.Publish)
+	logger.Gauge(fmt.Sprintf("%s.ack.count", queueInfo.Name), queueInfo.MessageStats.Ack)
+
+	// Can't use logger here, gauge only supports ints
```
Comment (nathanleiby) [inline main.go]: Derp, this sounds like a bug in the library, IMO. Let's make gauges (and counters?) take floats instead

**Comment by nathanleiby** [comment]
lgtm

File: main.go (hunk 2 above)
Comment (azylman) [inline main.go]: Should it take `interface{}`? Then the consumer can use `int`, `float`, `int64`, `uint`, etc.

Comment (nathanleiby) [inline main.go]: Ah, that might be best

**Pull request merged**